models/org.go

```diff
@@ -425,6 +425,25 @@ func GetOrgsByUserID(userID int64, showAll bool) ([]*User, error) {
 	return getOrgsByUserID(sess, userID, showAll)
 }
 
+// QueryUserOrgIDs returns a condition to return user's organization id
+func QueryUserOrgIDs(uid int64) *builder.Builder {
+	return builder.Select("team.org_id").
+		From("team_user").InnerJoin("team", "team.id = team_user.team_id").
+		Where(builder.Eq{"team_user.uid": uid})
+}
+
+// SimpleOrg represents a simple orgnization with only needed columns
+type SimpleOrg = User
```

Comment:
why not just use User directly?
for later refactors?!?

Reply:
To notice that the struct only have part fields filled. If we don't add this, we cannot differ the two Users.

Reply:
doesnt it make more sence then to create a new struct type?

Reply:
I have tried that, but we have to copy some functions from `User`.

Reply:
This is why most Java design moved to use POJOs for data with functionality moved to Services eg UserService to do actions on that data. 

Our data structs are too powerful & ultimately too tightly bound to the db interface. `modules/git` is another example and is why creating a second backend was so so hard.

For the moment don't call this "Simple" call it `MinimalUser` or something like that.